wpilibc/src/main/native/cpp/apriltag/AprilTagFieldLayout.cpp

```diff
@@ -26,36 +26,37 @@ AprilTagFieldLayout::AprilTagFieldLayout(std::string_view path) {
   wpi::json json;
   input >> json;
 
-  m_apriltags = json.at("tags").get<std::vector<AprilTag>>();
+  for (const auto& tag : json.at("tags").get<std::vector<AprilTag>>()) {
+    m_apriltags[tag.ID] = tag;
+  }
   m_fieldWidth = units::meter_t{json.at("field").at("width").get<double>()};
   m_fieldLength = units::meter_t{json.at("field").at("height").get<double>()};
 }
 
 AprilTagFieldLayout::AprilTagFieldLayout(std::vector<AprilTag> apriltags,
                                          units::meter_t fieldLength,
                                          units::meter_t fieldWidth)
-    : m_apriltags(std::move(apriltags)),
-      m_fieldLength(std::move(fieldLength)),
-      m_fieldWidth(std::move(fieldWidth)) {}
+    : m_fieldLength(std::move(fieldLength)),
+      m_fieldWidth(std::move(fieldWidth)) {
+    for (const auto& tag : apriltags) {
+      m_apriltags[tag.ID] = tag;
+    }
+}
 
 void AprilTagFieldLayout::SetAlliance(DriverStation::Alliance alliance) {
   m_mirror = alliance == DriverStation::Alliance::kRed;
 }
 
 std::optional<frc::Pose3d> AprilTagFieldLayout::GetTagPose(int ID) const {
-  Pose3d returnPose;
-  auto it = std::find_if(m_apriltags.begin(), m_apriltags.end(),
```

Comment:
`#include <algorithm>` can be removed now that `std::find_if()` is no longer used.